flo-runner/src/test/java/com/spotify/flo/context/FloRunnerTest.java

```diff
@@ -165,21 +164,21 @@ public void nonBlockingRunnerDoesNotBlock() throws Exception {
     }
 
     // If this file doesn't exist now, it's likely that runTask doesn't block
-    assertThat(Files.exists(happenedFile), is(false));
+    assertThat(happenedFile.exists(), is(false));
 
-    Files.write(latchFile, new byte[0]);
+    latchFile.createNewFile();
   }
 
   @Test
   public void blockingRunnerBlocks() throws IOException {
-    final Path file = temporaryFolder.newFile().toPath();
+    final File file = temporaryFolder.newFile();
```

Comment:
This change doesn't seem to be necessary. Maybe just to be consistent with other file usages?

Reply:
`Path` is not serializable.